modules/cnf-topology-aware-lifecycle-manager-policies-concept.adoc

```diff
@@ -6,9 +6,9 @@
 [id="talo-policies-concept_{context}"]
 = Update policies on managed clusters
 
-The {cgu-operator-first} remediates a set of `inform` policies for the clusters specified in the `ClusterGroupUpgrade` CR. {cgu-operator} remediates `inform` policies by making `enforce` copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding.
+The {cgu-operator-first} remediates a set of `inform` policies for the clusters specified in the `ClusterGroupUpgrade` CR. {cgu-operator} remediates `inform` policies by using {rh-rhacm}'s Selective Policy Enforcement feature to control the `remediationAction` specification in the `PolicyGenTemplate` resource.
```

Comment:
Not accurate. PolicyGenTemplate should not be mentioned. I removed "Selective Policy Enforcement feature" as well since this term is not mentioned in the ACM doc. 

Suggest the following
```suggestion
The {cgu-operator-first} remediates a set of `inform` policies for the clusters specified in the `ClusterGroupUpgrade` CR. {cgu-operator} remediates `inform` policies by controlling the `remediationAction` field of the policies through the placement binding `bindingOverrides.remediationAction` and `subFilter` fields. Each policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding.
```

The new fields we need (bindingOverrides.remediationAction and subFilter) from ACM is documented in [Policy Yaml Table](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/governance/governance#policy-yaml-table), which is included in "Policy overview" we linked below so we should be all set.

Reply:
Line 19
Change to "Once the timeout elapses, {cgu-operator} removes all its created resources to ensure that no further updates are made to clusters."

Reply:
I made a slight adjustment to your first suggestion. It clarifies that the `remediationAction` field is in the Policy resource, while the  `bindingOverrides.remediationAction` and `subFilter` specs are in the `PlacementBinding` resource. As I found this clearer. Hope that works for you too.